example.rs

```diff
@@ -0,0 +1,30 @@
+//! This is an example of how this crate could be usable
+
+#[macro_use] extern crate confy;
+
+#[derive(Confy)]
+struct MyConfig {
```

Comment:
a derive attr here could remove the need to pass `"my_app"` below. So something like

```rust
#[derive(Confy)]
#[confy(name="my_app")]
strucy MyConfig {
  // ... fields
}
```
and later, just `let cfg: MyConfig = confy::load()?` and `confy::create(MyConfig { /* ... */ })` etc.